Crane/app/src/main/java/androidx/compose/samples/crane/base/ExploreSection.kt

```diff
@@ -206,6 +199,13 @@ private fun ExploreImage(item: ExploreModel) {
                     .align(Alignment.Center),
             )
         }
+
+        Image(
```

Comment:
I've intentionally moved this after the `Image` for the loading state. This is so the "success" image is drawn on top of the placeholder if it's already in the memory cache since `AsyncImagePainter.state` is always `Loading` during the initial composition ([more info](https://coil-kt.github.io/coil/compose/#observing-asyncimagepainterstate)).